docs/migration-apiserver-to-crds.md

```diff
@@ -119,18 +119,37 @@ To test the mutation blocking feature, execute the following commands:
   ./service-catalog migration --action undeploy-blocker --service-catalog-namespace=default
   ```
 
-## Troubleshooting
+## Rollback
 
-In case your migration job failed, you can check its logs using the following command:
+In case you want to revert an upgrade you may use a `helm rollback` command which will restore the Service Catalog apiserver version.
```

Comment:
```suggestion
In case you want to revert the upgrade, use the `helm rollback` command which will restore the Service Catalog API Server version.
```